tests/foreman/cli/test_host.py

```diff
@@ -79,3 +82,205 @@ def test_create_libvirt_without_mac(self):
             u'puppet-proxy-id': self.puppet_proxy['id'],
             u'root-pass': host.root_pass,
         })
+
+
+class HostParameterTests(CLITestCase):
+    """Tests targeting host parameters"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create host to tests parameters for"""
+        super(HostParameterTests, cls).setUpClass()
+        cls.puppet_proxy = Proxy.list()[0]
```

Comment:
Can I suggest that we add the following assertion to make sure that the test can handle the possible situation where there's no default capsule (aka 'proxy')?

```python
assert len(Proxy.list()) > 0
```

If this were to fail without an assertion, then all tests would fail and it would be hard to determine what caused the failure. Jenkins would report that `setUp` failed with the traceback, but with an assertion we would be able to know right away what caused it.